uvloop/handles/process.pxd

```diff
@@ -8,7 +8,7 @@ cdef class UVProcess(UVHandle):
         object _preexec_fn
         bint _restore_signals
 
-        set _fds_to_close
+        list _fds_to_close
```

Comment:
@fantix I think there's a proper `list[int]` data type in cython for this but I don't know how to use it